Yafc.Parser/Data/FactorioDataDeserializer.cs

```diff
@@ -132,12 +132,17 @@ public Project LoadData(string projectPath, LuaTable data, LuaTable prototypes,
         DeserializePrototypes(raw, "fluid", DeserializeFluid, progress, errorCollector);
         progress.Report(("Loading", "Loading recipes"));
         DeserializePrototypes(raw, "recipe", DeserializeRecipe, progress, errorCollector);
+        progress.Report(("Loading", "Loading locations"));
+        DeserializePrototypes(raw, "planet", DeserializeLocation, progress, errorCollector);
+        DeserializePrototypes(raw, "space-location", DeserializeLocation, progress, errorCollector);
+        rootAccessible.Add(GetObject<Location>("nauvis"));
```

Comment:
Can we get this value from the mo-pack data somehow? Maybe some 'default setting on one of the locations, or something?

I can imagine there will be mods (or even a 2.1 release...? 😅) that let you start on another planet (I think I remembered reading (some FFF maybe) telling that is would be possible to start on any of these 4 ('basic)planets and finish the game?)

Reply:
I think I remember that too, now that you mention it. I can't find anything that looks promising, though. I dumped the data.raw table, and didn't find `"nauvis"` anywhere that looked interesting. I also didn't see anything in CharacterPrototype or SurfacePropertyPrototype.

Reply:
We can leave it for now, until the actual need arises. I was just wondering.